core/src/components/spinner/test/resize/spinner.e2e.ts

```diff
@@ -0,0 +1,33 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+/**
+ * The resize behavior does not vary across directions or modes.
+ */
+configs({ directions: ['ltr'], modes: ['md'] }).forEach(({ title, screenshot, config }) => {
+  test.describe(title('spinner: resize'), () => {
+    test.beforeEach(async ({ page }) => {
+      await page.setViewportSize({ width: 320, height: 340 });
+    });
+    test('should not have visual regressions', async ({ page }) => {
+      await page.setContent(
+        `
+        <ion-content id="content">
```

Comment:
It doesn't look like the spinners are being resized at all. Is this intentional?

Reply:
Oops! Nope, I hadn't had a chance to look at the screenshots yet from CI, and I can't run this test file locally for some reason (but others are fine) 🤔